src/index.test.ts

```
@@ -0,0 +1,137 @@
+      expect(Array.isArray(result)).toBe(true);
+      expect(result.length).toBeGreaterThanOrEqual(0); // May be empty if no CCM networks exist
+    });
+  });
+
+  describe('getNetworkName', () => {
+    it('should be defined and callable', () => {
+      expect(getNetworkName).toBeDefined();
+      expect(typeof getNetworkName).toBe('function');
+    });
+
+    it('should return network name for a given alias', () => {
```

ESLint

[error] Replace `'should·return·network·name·for·a·given·alias'` with `"should·return·network·name·for·a·given·alias"`

(prettier/prettier)